### PR TITLE
133 neighborhood page add section to show members

### PR DESCRIPTION
**Pull request by ecemgo**
I've created a component called 'ShowMembers'. However, I wrote it with jsx instead tsx as I have no idea about the syntax of typescript. Sorry for this. And, I added css file for it.

### CODE REVIEW

File: apps/frontend/src/components/ShowMembers/ShowMembers.jsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+function App() {
+  return (
+    <div>
+      <UserCircleStack />
+    </div>
+  );
+}
+
+function UserCircleStack() {
```
Comment (radu-constantin) [inline apps/frontend/src/components/ShowMembers/ShowMembers.jsx:15]: This component should be able to take usernames as **props**, just like you did with the **UserCircle** component!

**Review by radu-constantin — CHANGES_REQUESTED** [review]
Excellent work Ecem! You're getting better and better at React. Keep up the good work!
I've added two comments that you should check out.
Don't worry about typescript, I will convert it later on and integrate the component in the neighborhood page.

File: apps/frontend/src/components/ShowMembers/ShowMembers.jsx (hunk 1 above)
Comment (ecemgo) [reply apps/frontend/src/components/ShowMembers/ShowMembers.jsx]: I changed it as you mentioned.

File: apps/frontend/src/components/ShowMembers/ShowMembers.jsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+function UserCircle({ username }) {
+  const firstTwoLetters = username.slice(0, 2).toUpperCase();
+
+  return (
+    <div className="circle" style={{ style: "--i: 1" }}>
+      {firstTwoLetters}
+    </div>
+  );
+}
```
Comment (radu-constantin) [inline apps/frontend/src/components/ShowMembers/ShowMembers.jsx:35]: Usually, the common practice when you work with React is that each component should have its own file, I would recommend that you create a separate folder and file for this component.

File: apps/frontend/src/components/ShowMembers/ShowMembers.jsx [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  const usernames = ["radu", "antonina", "ecem", "maria", "mike", "john"];
+
+  return (
+    <div>
+      <UserCircleStack usernames={usernames} />
+    </div>
+  );
+}
+
+function UserCircleStack({ usernames }) {
```
Comment (radu-constantin) [inline apps/frontend/src/components/ShowMembers/ShowMembers.jsx:15]: The file and folder name should have the same name as the exported component. In this case, your file name and folder name should be 'UserCircleStack'

**Review by radu-constantin — CHANGES_REQUESTED** [review]
I've added some comments that I forgot to mention yesterday

File: apps/frontend/src/components/ShowMembers/ShowMembers.jsx (hunk 3 above)
Comment (ecemgo) [reply apps/frontend/src/components/ShowMembers/ShowMembers.jsx:15]: Sorry for doing it so late. I did as you mentioned. I hope it's true. :)

**Review by radu-constantin — APPROVED** [review]
Good job Ecem! Approved

**Pull request merged**